packages/components/src/components/modal/_modal.scss

```diff
@@ -140,8 +211,28 @@
     }
   }
 
-  .#{$prefix}--modal-content > * {
-    @include type-style('body-long-01');
+  .#{$prefix}--modal-content {
+    > * {
+      @include type-style('body-long-01');
```

Comment:
would this override type styles for all children in the modal content area? I may have missed where this requirement is in the specs

Reply:
Good question, I don't know (though I guess you are right). This part has no change from the before-change code.